extensions/VisualEditor/wikia/modules/ve/ce/ve.ce.WikiaBlockImageNode.js

```diff
@@ -0,0 +1,33 @@
+/*!
+ * VisualEditor ContentEditable WikiaBlockImageNode class.
+ *
+ * @copyright 2011-2013 VisualEditor Team and others; see AUTHORS.txt
+ * @license The MIT License (MIT); see LICENSE.txt
+ */
+
+/**
+ * VisualEditor ContentEditable Wikia image node.
+ *
+ * @class
+ * @extends ve.ce.WikiaBlockMediaNode
+ *
+ * @constructor
+ * @param {ve.dm.WikiaBlockImageNode} model Model to observe
+ * @param {Object} [config] Config options
+ */
+ve.ce.WikiaBlockImageNode = function VeCeWikiaBlockImageNode( model, config ) {
```

Comment:
I guess function VeCeWikiaBlockImageNode (model, config) { would be more guidelines-compliant ;)